server/filestore.go

```diff
@@ -202,6 +202,7 @@ type fileStore struct {
 	firstMoved  bool
 	ttls        *thw.HashWheel
 	ttlseq      uint64 // How up-to-date is the `ttls` THW?
+	markers     *stree.SubjectTree[struct{}]
```

Comment:
This could be heavy for lighter use cases, but understand why this might be more applicable the `[]string` but I would start with that tbh.

Reply:
Yes, is rather a premature optimisation I suppose. I can change it to `[]string` for now.